CMakeLists.txt

```
@@ -168,11 +177,14 @@ else()
     message(FATAL_ERROR "Could not find libraries for outcome.")
 endif()
 
-find_package(Catch2 3.8.0 REQUIRED)
-if(Catch2_FOUND)
-    message(STATUS "Found Catch2 ${Catch2_VERSION}.")
-else()
-    message(FATAL_ERROR "Could not find libraries for Catch2.")
+if(SPIDER_ENABLE_TESTS)
+    message(WARNING "Building tests: ${SPIDER_ENABLE_TESTS}")
+    find_package(Catch2 3.8.0 REQUIRED)
+    if(Catch2_FOUND)
+        message(STATUS "Found Catch2 ${Catch2_VERSION}.")
+    else()
+        message(FATAL_ERROR "Could not find libraries for Catch2.")
+    endif()
```

```suggestion
message(STATUS "Found Catch2 ${Catch2_VERSION}.")
```

Since `Catch2` is `REQUIRED` CMake will already error out if it is not found. This makes the else block dead code and the if statement redundant.
(I know this is new compared to existing code, sorry.)